hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ratis/RatisCommands.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Move the code inside, `ozone/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin`